docs/user/document/references-usage.en.md

```diff
@@ -1,3 +1,11 @@
 # References - Usage
 
-* There should be one reference item that provides a canonical URL for this document, i.e. it has the category `self`, the `url` starts with `https://` and ends with a valid filename for the CSAF document according to the rules in section 5.1. of the specification.
+All links that have any relevance in the overall context of the document should be listed here.
+This could include for example original advisories or advisories of the manufacturer or especially the coordinator.
+If other advisories (especially from suppliers) are referenced, they should be listed here.
+
+There should be one reference item that provides a canonical URL for this document, i.e. it has the category `self`, the `url` starts with `https://` and ends with a valid filename for the CSAF document according to the rules in section 5.1. of the specification.
+
+It is recommended to link the recommended cybersecurity best practices of the team where advisories are found.
```

Comment:
advisories => singular as it is only about the current (makes sense esp. for coordinators).
team => organization

Reply:
updated as suggested and rephrased as some meaning was lost in translation, please check

Reply:
Not 100% but much better.